moviepy/audio/fx/audio_volume_lr.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+from moviepy.decorators import audio_video_fx
+
+
+@audio_video_fx
+def audio_volume_lr(clip, left=1, right=1):
```

Comment:
I think that the name should be in line with the current afx `volumex`. So have `volumex` and `stereo_volumex`? Or name this `audio_stereo_volume` and rename `volumex` to `audio_volume`? I'm not a massive fan of the name `volumex`, but I didn't change it in #1170.

Reply:
I love your suggestion `volumex` -> `audio_volume` and this as `audio_stereo_volume` because I think the same about `volumex` :+1: